### ISSUE TITLE
Does OCC support POSIX path?

### ISSUE THREAD

**Issue by 3oheicrw** [LADSoft/OrangeC#722]
I wanted to use OCC in Cygwin and MSYS2. It will be used with Makefile (GNU Make). There are POSIX paths in these Makefiles (use the forward slash and include location like `/usr/include` that doesn't make sense to a pure Win32 application). It seems OCC itself is a pure Win32 application so of course it doesn't support this kind of path. But what if I build my own version of OCC with Cygwin GCC/MSYS2 GCC (non-MinGW)? This way OCC will link with cygwin-1.dll or msys-2.0.dll if use MSYS2. Will it make OCC magically able to work with paths like this? Thanks.

**Comment by GitMensch** [comment]
You only have to ensure that the paths are valid under Windows without translation so for example if you compile under /mingw/home and this is x:\mingw\home in reality, then there is no problem, same if you use relative paths.

**Comment by 3oheicrw** [comment]
> You only have to ensure that the paths are valid under Windows without translation so for example if you compile under /mingw/home and this is x:\mingw\home in reality, then there is no problem, same if you use relative paths.

I mean cygwin/msys2 specific paths that only has meanings in cygwin/msys2 shell like `/usr/include`. The Windows path for it should be `C:\Cygwin64\usr\include` or `C:\msys64\usr\include`

**Comment by GitMensch** [comment]
If you compile OCC on that environment and don't compile it again with itself, then yes, that should work as the C library does the translation.

**Comment by LADSoft** [comment]
yeah my next thing is going to be to try to get it to compile with the latest version of mingw64 and fix whatever the problem was with clang
